packages/rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -26,6 +26,35 @@ registerStyles('vaadin-rich-text-editor', richTextEditorStyles, { moduleId: 'vaa
 
 const Quill = window.Quill;
 
+// Workaround for text disappearing when accepting spellcheck suggestion
+// See https://github.com/quilljs/quill/issues/2096#issuecomment-399576957
+const Inline = Quill.import('blots/inline');
+
+class CustomColor extends Inline {
+  constructor(domNode, value) {
+    super(domNode, value);
+
+    // Map <font> properties
+    domNode.style.color = domNode.color;
+
+    const span = this.replaceWith(new Inline(Inline.create()));
+
+    span.children.forEach((child) => {
+      if (child.attributes) child.attributes.copy(span);
+      if (child.unwrap) child.unwrap();
+    });
+
+    this.remove();
```

Comment:
If you comment out this line, you'll notice that the DOM node is originally a `<font>` element, inserted by the browser spellcheck functionality exactly as described in https://github.com/quilljs/quill/issues/2096#issuecomment-399516461.


<img width="902" alt="Screenshot 2023-09-18 at 15 54 59" src="https://github.com/vaadin/web-components/assets/10589913/1811b0a0-149f-4322-ab68-87b2f92e1f9e">